docs/fundamentals/runtime-libraries/snippets/System/String/Overview/csharp/index11.cs

```diff
@@ -13,8 +13,7 @@ public static void Main()
          if (Char.IsPunctuation(s1[ctr]) | Char.IsWhiteSpace(s1[ctr]))
             nWords++;              
       }
-      Console.WriteLine("The sentence\n   {0}\nhas {1} words.",
-                        s1, nWords);                                                                     
+      Console.WriteLine($"The sentence\n   {s1}\nhas {nWords} words.");                                                                     
```

Comment:
I'd consider raw string literals here:

```suggestion
      Console.WriteLine($"""
      The sentence
         {s1}
       has {nWords} words.
       """);                                                                     
```